client/src/App.jsx

```diff
@@ -2,16 +2,17 @@ import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import { CssBaseline } from '@material-ui/core';
 import Navigation from './components/Navigation/Navigation';
-import Login from './pages/SignIn/SignIn';
+import SignIn from './pages/SignIn/SignIn';
 
 function App() {
   return (
     <>
       <CssBaseline />
       <Navigation />
+      <SignIn />
```

Comment:
This line can be removed because a better approach is to keep it in the Switch/Route as it is on line 15.

I would suggest looking into the `NavigationDrawer` component and add the page there inside of the `navItems` array since the shop won't be defaulting to the **Sign In** page.

Reply:
Done. Thank you!